main.py

```diff
@@ -0,0 +1,60 @@
+from envs import Env4InRow
+
+player1 = "d"
+player2 = "n"
+width = 8
+height = 6
+game = Env4InRow(player1, player2, (height, width))
+
+
+def play(player, action):
+    if not game.apply_action(player, action):
+        print("Wrong action!")
+    status = game.player_status(player1)
+    if status == 1:
+        print("Player1 won!")
+    if status == -1:
+        print("Player1 lost!")
+    if game.is_terminal_state():
+        print("Game over(c)! Board is full!")
+
+
+game.render()  # Test initial rendering
```

Comment:
For testing use the `pytest` utility - easy and great for testing and supported by PyCharm.